spec/zoneddatetime.html

```diff
@@ -1196,8 +1196,11 @@ <h1>
         1. Else,
           1. Let _timeZoneID_ be ? ToString(_timeZone_).
           1. Let _timeZoneString_ be the string-concatenation of the code unit 0x005B (LEFT SQUARE BRACKET), _timeZoneID_, and the code unit 0x005D (RIGHT SQUARE BRACKET).
-        1. Let _calendarID_ be ? ToString(_zonedDateTime_.[[Calendar]]).
-        1. Let _calendarString_ be ! FormatCalendarAnnotation(_calendarID_, _showCalendar_).
+        1. If _showCalendar_ is *"never"*, then
+          1. Let _calendarString_ be the empty String.
+        1. Else,
+          1. Let _calendarID_ be ? ToString(_zonedDateTime_.[[Calendar]]).
+          1. Let _calendarString_ be ! FormatCalendarAnnotation(_calendarID_, _showCalendar_).
```

Comment:
Is it worth making an AO for these repeated steps?

Reply:
Sure, I've done so.

Reply:
I meant the full if/else - is something unpushed?

Reply:
I accidentally pushed it to my own fork instead of anba's, so it didn't update this PR. Should be updated now.